keyboards/iris/keymaps/s1carii/keymap.c

```diff
@@ -0,0 +1,147 @@
+#include "iris.h"
```

Comment:
Could you replace the includes with just `#include QMK_KEYBOARD_H`, as that's all that's needed here.